lineage/data/Lineage_collections.py

```diff
@@ -8,38 +8,42 @@
 desired_num_states = 2
 E = [StateDistribution() for _ in range(desired_num_states)]
 
-
-def popout_single_lineages(lineages):
-    """ To remove lineages with cell numbers <= 5. """
-    trimed_lineages = []
-    for cells in lineages:
-        if len(cells) < 5:
-            pass
-        else:
-            trimed_lineages.append(cells)
-    assert len(trimed_lineages) > 0
+def popout_single_lineages(lineages):	
```

Comment:
Do we use this?

Reply:
It is used for figure 1. I can go through the files and find proper lineages, but for now the indexes we use to plot those exact lineages.